jigasi.yml

```diff
@@ -24,6 +26,12 @@ services:
             - JIGASI_BREWERY_MUC
             - JIGASI_PORT_MIN
             - JIGASI_PORT_MAX
+            - ENABLE_TRANSCRIPTIONS
+            - JIGASI_TRANSCRIBER_ADVERTISE_URL
+            - JIGASI_TRANSCRIBER_RECORD_AUDIO
+            - JIGASI_TRANSCRIBER_SEND_TXT
+            - GOOGLE_APPLICATION_CREDENTIALS
+            - LETSENCRYPT_DOMAIN
```

Comment:
is this one still needed then?

Reply:
oh, missed it. I removed line `- LETSENCRYPT_DOMAIN`